src/Core/src/Handlers/WebView/WebViewHandler.Android.cs

```diff
@@ -25,7 +25,7 @@ protected override AWebView CreatePlatformView()
 		{
 			var platformView = new MauiWebView(this, Context!)
 			{
-				LayoutParameters = new LayoutParams(LayoutParams.MatchParent, LayoutParams.WrapContent)
+				LayoutParameters = new LayoutParams(LayoutParams.MatchParent, LayoutParams.MatchParent)
```

Comment:
Does this cause any weirdness now if you have the WebView in a VerticalStackLayout?

Reply:
Not that I could see. I tried a piece of html that was 1 line and it seemed to size the web view to be 1 line. What was the reason to change this from forms?